src/middleware/src/storage/webstorage.js

```diff
@@ -4,7 +4,6 @@ import keyBy from 'lodash/keyBy';
 import merge from 'lodash/merge';
```

Comment:
I don't agree. I think if things are closely enough related then it is perfectly fine to have multiple classes in one file.

Reply:
But this makes navigation between files so much more difficult. If I want to check something in `LocalStorageAdapter` class, I need to use "find in files". Or know that it is in an arbitrarily named file. Right now, "webstorage.js" would be the last thing that comes to mind, when looking for a storage adapter. I'd try "localstorage", "storage", "adapter" and will not find it.

And this is the case with many more files/classes.

Reply:
I like the way it is and would not like to see it changed. I like that all web storage adapters are grouped into the same file since they are related.

Reply:
They'd still be grouped, but in a folder. Has the same benefits, without the drawbacks.

Anyways, your call.